pkg/virt-controller/watch/node.go

```diff
@@ -23,6 +23,10 @@ import (
 	"kubevirt.io/kubevirt/pkg/log"
 )
 
+const (
+	NodeUnresponsiveVirtualMachineInstanceReason = "NodeUnresponsive"
```

Comment:
Just a very little thing, but it catched my eyes: I think it's quite common to start these kind of const names with WHERE it's used (e.g. `VirtualMachineInstanceStatusReason`) and then the actual reason (`NodeUnresponsive`), so this would be `VirtualMachineInstanceStatusReasonNodeUnresponsive`. And because this could potentially be a reasonable status reason for other resources too, I would be fine with skipping `VirtualMachineInstance`, which also makes the name shorter. See e.g. https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L613

Reply:
Done.